CONTRIBUTING.md

```diff
@@ -5,34 +5,39 @@
 You must follow these guidelines if you wish to contribute to the PocketMine-MP code base, or participate in issue tracking.
 
 ## I have a question
-* For questions, please refer to the _#pocketmine_ or _#mcpedevs_ IRC channel on Freenode. There is a [WebIRC](http://webchat.freenode.net?channels=pocketmine,mcpedevs&uio=d4) if you want.
+* For questions, please refer to the _#pmmp_ or _#pocketmine_ IRC channel on Freenode. There is a [WebIRC](http://webchat.freenode.net?channels=pmmp,pocketmine&uio=d4) if you do not want to install an IRC client.
```

Comment:
I believe the link to `#mcpedevs` served more like a hidden advertisement. I really don't think anyone who wanted to create an issue on PocketMine-MP should actually be redirected to `#mcpedevs`, but would only make more people talking at the wrong channel.